e2e/api/test/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html class="no-js nimbus pf-m-redhat-font">
+  <head>
+    <!-- SPA_HEAD -->
+    <!--#include virtual="/init.html" -->
```

Comment:
Read more in [FAQ](/spaship/spaship/wiki/FAQ)
```suggestion
    <!--#include virtual="/.include/init.html" -->
```

Reply:
fixed